src/components/DashboardSpeakers/DashboardSpeakersPresentation.js

```diff
@@ -1,11 +1,12 @@
-import React from 'react'
+import React, { Fragment } from 'react'
 import PropTypes from 'prop-types'
 import { Link } from 'react-router'
 
 // FPCC
 import DashboardLanding from 'components/DashboardLanding'
 import DashboardTable from 'components/DashboardTable'
 import getIcon from 'common/utils/getIcon'
+import InfiniteLoadBtn from 'components/InfiniteLoadBtn/InfiniteLoadBtn'
 
 function DashboardSpeakersPresentation({
   queryResponse,
```

Comment:
Just for clarity and to stick with our convention, it is best to rename`queryResponse` to `infiniteQueryResponse` in the data layer and pass it through as such. It helps to make it clear that this is an infinite query response.

Reply:
ok, I was wondering about that. I had it like that but then changed it for some reason